go-controller/pkg/cni/cni.go

```diff
@@ -136,8 +136,11 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, podLister corev1listers.PodL
 	if err != nil {
 		return nil, fmt.Errorf("failed to get pod annotation: %v", err)
 	}
-	if err := pr.checkOrUpdatePodUID(podUID); err != nil {
-		return nil, err
+	pod, err := getPod(podLister, kclient, pr.PodNamespace, pr.PodName)
+	if !IsStaticPod(pod) && err == nil {
+		if err := pr.checkOrUpdatePodUID(podUID); err != nil {
```

Comment:
NIT: remove the ':' from ':='

Reply:
im declaring and assigning the variable pod here so I need the := I think

Reply:
ah, ack.